packages/cli/generators/app/templates/index.js.ejs

```diff
@@ -6,8 +6,8 @@ if (require.main === module) {
   // Run the application
   const config = {
     rest: {
-      port: +process.env.PORT || 3000,
-      host: process.env.HOST || 'localhost',
+      port: +(process.env.PORT || 3000),
+      host: process.env.HOST,
```

Comment:
For the endpoint in the docker container to be accessed from the host operating system, the binding ip/host should be from `HOST` env var or `undefined` (all interfaces).